go.mod

```diff
@@ -22,23 +22,22 @@ require (
 	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
 	github.com/prometheus/procfs v0.0.4 // indirect
 	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/pflag v1.0.3
+	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0 // indirect
-	golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392 // indirect
-	golang.org/x/net v0.0.0-20191003171128-d98b1b443823 // indirect
 	golang.org/x/sys v0.0.0-20200113162924-86b910548bc1 // indirect
 	google.golang.org/appengine v1.6.2 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/api v0.17.1
+	k8s.io/apimachinery v0.17.1
 	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/klog v1.0.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20190918143330-0270cf2f1c1d
 	sigs.k8s.io/controller-runtime v0.3.0
 )
 
 // Pinned to kubernetes-1.15.4
 replace (
+
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200117162508-e7ccdda6ba67
```

Comment:
For some reason, even if I change the dependency in the "require" section (L17), it always gets reset by `go mod tidy` :thinking:

Reply:
that is expected and what the replace is needed for

Reply:
@davidffrench why wouldn't it let us just specify an expected version in the `require` section? I thought the `replace` was more for transitive dependency versions, but that's just based on what I expect it to be for, rather than having read it somewhere.

Reply:
Maybe it is possible but this is what I have noticed from my own use. A replace is about pinning to a specific version so even if I run `go get <module>@latest`, the require will be updated but the replace is still pinning it. `go mod tidy` can also change the require but I don't know under what circumstances.

Reply:
Austin said he had success in fixing this issue, there's a change to go.mod in this PR https://github.com/integr8ly/cloud-resource-operator/commit/47646dbdff784f7dc09c0cd73dad7880f29892d8